### PR TITLE
add unknown modifications as chemmod

### PR DESCRIPTION
**Pull request by timosachsenberg**
Improved support for unknown modification export: 
https://github.com/OpenMS/OpenMS/issues/4835

### CODE REVIEW

**Comment by hroest** [comment]
any chance you can add a test for this?

File: src/openms/source/FORMAT/MzTab.cpp [hunk 1]
```diff
@@ -3027,10 +3053,20 @@ Not sure how to handle these:
         if (!is_fixed)
         {
           String unimod = res_mod.getUniModAccession();
+
```
Comment (cbielow) [inline src/openms/source/FORMAT/MzTab.cpp:3056]: this whole block looks awfully like copy and pasted a couple of times. Can you make that a function?

Comment (cbielow) [reply src/openms/source/FORMAT/MzTab.cpp:3056]: :)

Comment (timosachsenberg) [reply src/openms/source/FORMAT/MzTab.cpp:3056]: when I am through with the other issues :P

**Pull request merged**